### PR TITLE
Added API1600 support to hypervisor managers

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added API1600 support to hypervisor managers

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/shared_samples/hypervisor_manager.rb [hunk 1]
```diff
@@ -29,6 +31,8 @@
 # api_version = 1000 & variant = Synergy to OneviewSDK::API1000::Synergy::HypervisorManager
 # api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::HypervisorManager
 # api_version = 1200 & variant = Synergy to OneviewSDK::API1200::Synergy::HypervisorManager
+# api_version = 1600 & variant = C7000 to OneviewSDK::API1200::C7000::HypervisorManager
```
Comment (chebroluharika) [inline examples/shared_samples/hypervisor_manager.rb:34]: It should be :API1600::C7000::HypervisorManager

File: examples/shared_samples/hypervisor_manager.rb [hunk 2]
```diff
@@ -29,6 +31,8 @@
...
+# api_version = 1600 & variant = Synergy to OneviewSDK::API1200::Synergy::HypervisorManager
```
Comment (chebroluharika) [inline examples/shared_samples/hypervisor_manager.rb:35]: same as above

File: examples/shared_samples/hypervisor_manager.rb (hunk 1 above)
Comment (VenkateshRavula) [reply examples/shared_samples/hypervisor_manager.rb]: done

File: examples/shared_samples/hypervisor_manager.rb (hunk 2 above)
Comment (VenkateshRavula) [reply examples/shared_samples/hypervisor_manager.rb]: done

**Pull request merged**